bootstrap/kubeadm/api/v1beta1/kubeadmconfig_types_test.go

```diff
@@ -1,159 +0,0 @@
-/*
```

Comment:
why are these tests being dropped?

Reply:
They are not dropped, just moved to `bootstrap/kubeadm/api/v1beta1/kubeadmconfig_webhook_test.go` in 119d616bee37915bc1f626c40ead9e3aee753deb.

Reply:
I think this can be resolved.